tools/cockpit.spec

```diff
@@ -427,8 +427,17 @@ cluster. Installed on the Kubernetes master. This package is not yet complete.
 
 %if %{defined gitcommit}
 
+%package playground
```

Comment:
Why are we splitting this out? Is it not possible to force install cockpit-test-assets?